homeassistant/components/roku/media_player.py

```diff
@@ -238,16 +238,33 @@ async def search(self, keyword):
         """Emulate opening the search screen and entering the search keyword."""
         await self.coordinator.roku.search(keyword)
 
+    async def async_get_browse_image(
+        self, media_content_type, media_content_id, media_image_id=None
+    ):
+        """Fetch media browser image to serve via proxy."""
+        if media_content_type == MEDIA_TYPE_APP and media_content_id:
+            image_url = self.coordinator.roku.app_icon_url(media_content_id)
+            result = await self._async_fetch_image(image_url)
+            if result == (None, None):
+                _LOGGER.debug("Error retrieving proxied image from %s", image_url)
```

Comment:
We should move this error to the HTTP View. The fact that we need to proxy an image and then get no response will always be a warning.

Reply:
If it is not supposed to be proxies, we shouldn't generate a proxy url.